Controller/Component/WizardComponent.php

```diff
@@ -157,6 +157,13 @@ class WizardComponent extends Component {
 	public $nestedViews = false;
 
 /**
+ * Holds the root of the session key for data storage.
+ *
+ * @var string
+ */
+	public $sessionRootKey = 'Wizard';
```

Comment:
This can be an option in the component options array 
```
public $components = array(
    'Wizard.Wizard' => array(
        'sessionRootKey' => 'MyWizard',
    ),
); 
```
or it can be set in the controller's beforeFilter()
```
$this->Wizard->sessionRootKey = 'MyWizard';
```